src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/GlobalExceptionHandler.kt

```diff
@@ -175,17 +190,10 @@ class GlobalExceptionHandler(
   }
 
   @ExceptionHandler(Exception::class)
-  fun handleException(e: Exception): ResponseEntity<ErrorResponse?>? {
+  fun unexpectedExceptionHandler(e: Exception, request: WebRequest): ResponseEntity<Any>? {
```

Comment:
This was the previous exception handler that was being used for database errors. I've refactored it so that it's response body content is equally vague, but we still log the exception.

I also had to rename the method because I needed to add `request: WebRequest` to the method args, but there is already a method `handleException(e: Exception, request: WebRequest)` in the superclass that this class extends.

Reply:
Nice – makes sense. Side thought, but it might be worth setting up Sentry to log / track these unexpected errors at some point.